cranelift/filetests/filetests/isa/riscv64/call.clif

```diff
@@ -396,11 +396,13 @@ block0(v0: i128, v1: i64):
 
 ; VCode:
 ; block0:
-;   mv a0,a1
+;   mv a0,a2
+;   mv a1,a0
 ;   ret
 ; 
 ; Disassembled:
 ; block0: ; offset 0x0
+;   ori a2, a0, 0
 ;   ori a0, a1, 0
 ;   ret
```

Comment:
Here the VCode pretty-printing changed without changing the disassembly listing. The same instruction was originally printed as `mv a0,a1` and now is printed as `mv a1,a0`, but in both cases it's disassembled as `ori a0, a1, 0`. Any idea what's going on there? I don't see any Rust-side changes that explain this. There are more cases like this below.